saleor/product/utils/product_variants.py

```diff
@@ -0,0 +1,21 @@
+from ...graphql.core.utils import from_global_id_strict_type
```

Comment:
I'm not sure is `product_variants.py` is good name for this function?

Reply:
Maybe we don't need a new module for these two functions? We can move them to `product/utils/__init__.py`.